### PR TITLE
Atlassian.Sourcetree version 3.4.5

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18015)

### CODE REVIEW

File: manifests/a/Atlassian/Sourcetree/3.4.5/Atlassian.Sourcetree.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Atlassian.Sourcetree
+PackageVersion: 3.4.5
```
Comment (KaranKad) [inline manifests/a/Atlassian/Sourcetree/3.4.5/Atlassian.Sourcetree.installer.yaml:5]:
```suggestion
PackageVersion: 3.4.5.0
```
You will need to edit the folder name as well from `manifests/a/Atlassian/Sourcetree/3.4.5/` to `manifests/a/Atlassian/Sourcetree/3.4.5.0/`
image

File: manifests/a/Atlassian/Sourcetree/3.4.5/Atlassian.Sourcetree.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Atlassian.Sourcetree
+PackageVersion: 3.4.5
```
Comment (KaranKad) [inline manifests/a/Atlassian/Sourcetree/3.4.5/Atlassian.Sourcetree.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.4.5.0
```

File: manifests/a/Atlassian/Sourcetree/3.4.5/Atlassian.Sourcetree.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Atlassian.Sourcetree
+PackageVersion: 3.4.5
```
Comment (KaranKad) [inline manifests/a/Atlassian/Sourcetree/3.4.5/Atlassian.Sourcetree.yaml:5]:
```suggestion
PackageVersion: 3.4.5.0
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Comment by KaranKad** [comment]
You will need to edit folder from `manifests/a/Atlassian/Sourcetree/3.4.5/` to `manifests/a/Atlassian/Sourcetree/3.4.5.0/`

**Comment by krystofkrticka** [comment]
@KaranKad working on it

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**